src/xai_grok/errors/model_not_found_error.py

```diff
@@ -0,0 +1,4 @@
+class ModelNotFoundError(Exception):
+
+    def __init__(self, message: str):
+        super().__init__(message)
```

Comment:
The indentation in the file is incorrect. The `__init__` method should be indented to be part of the `ModelNotFoundError` class, otherwise it will be treated as a standalone function.